.github/workflows/release.yaml

```diff
@@ -52,7 +52,8 @@ jobs:
         REGISTRY_USERNAME: ${{ secrets.REGISTRY_USERNAME }}
         IMAGE_HOST: quay.io
         IMAGE: shipwright/shipwright-operator
-      run: make release
+      run: |
+        TAG="${{ github.events.input.release }}" make release
```

Comment:
Does it make sense to move `TAG` up to the other environment variables, just for consistency?

Reply:
Excellent idea. Also updated nightly.yaml to do the same. 👍